pkg/datasource/csvsource/csv_kline_reader_test.go

```diff
@@ -0,0 +1,155 @@
+package csvsource
+
+import (
+	"encoding/csv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+	"github.com/c9s/bbgo/pkg/types"
+)
+
+var assertKLineEq = func(t *testing.T, exp, act types.KLine) {
+	assert.Equal(t, exp.StartTime, act.StartTime)
+	assert.True(t, exp.Open == act.Open)
+	assert.True(t, exp.High == act.High)
+	assert.True(t, exp.Low == act.Low)
```

Comment:
fixedpoint.Value could be a struct, it's better to use a.Compare(b)

Reply:
still wondering how this op is different it seems exactly the same?
func (x Value) Compare(y Value) int {
	if x > y {
		return 1
	} else if x == y {
		return 0
	} else {
		return -1
	}
}

Reply:
== is the only operator that works. struct with same field value.